cmd/controlplane/external_webhooks.go

```diff
@@ -77,6 +77,10 @@ func (o *externalWebhooksServerOptions) run(ctx context.Context) error {
 		return fmt.Errorf("error creating Kubernetes client: %w", err)
 	}
 
+	if err = kargoapi.AddToScheme(cluster.GetClient().Scheme()); err != nil {
+		return fmt.Errorf("failed to add custom kargo type to runtime scheme: %w", err)
```

Comment:
This error message seems a bit off. The failure wasn't in adding _a_ custom type, but an entire API. Everywhere else in the code where an error similar to this is handled, the message looks like this:

https://github.com/akuity/kargo/blob/cc0bed76da157a1a89c6ccddd74c98cc1acacabf/cmd/controlplane/controller.go#L154-L159

Possible to get a follow-up on this PR?